src/Component.php

```diff
@@ -2,15 +2,94 @@
 
 declare(strict_types=1);
 
-namespace MyComponent;
+namespace ProjectMigrationTool;
 
 use Keboola\Component\BaseComponent;
+use ProjectMigrationTool\Configuration\Config;
+use ProjectMigrationTool\Configuration\ConfigDefinition;
+use ProjectMigrationTool\Snowflake\Command;
+use ProjectMigrationTool\Snowflake\ConnectionFactory;
 
 class Component extends BaseComponent
 {
     protected function run(): void
     {
-        // @TODO implement
+        $databases = $this->getConfig()->getDatabases();
+
+//        Create database connections
+        $this->getLogger()->info('Connecting to databases.');
+        $sourceSnflkConnection = ConnectionFactory::create('source');
+        $migrateSnflkConnection = ConnectionFactory::create('migrate');
+        $destinationSnflkConnection = ConnectionFactory::create('destination');
+
+//        Switch to accountadmin role
+        Command::useRole($sourceSnflkConnection, 'ACCOUNTADMIN');
+        Command::useRole($migrateSnflkConnection, 'ACCOUNTADMIN');
+        Command::useRole($destinationSnflkConnection, 'ACCOUNTADMIN');
+
+//        Cleanup destination account
+        Command::cleanupProject($destinationSnflkConnection);
+
+//        Create KEBOOLA_STORAGE role in destination anflk account
```

Comment:
Role nemusí být KEBOOLA_STORAGE - může být třeba SAPI (v US) - měl bys to udělat podle toho, co je v tom source účtu

Reply:
řeší se https://github.com/keboola/project-migration-tool/pull/1#discussion_r1086810498